### PR TITLE
feat: added info icon appearing from below with smooth animation.

### PR DESCRIPTION
**Pull request by Kamil-Drozdz**

### CODE REVIEW

File: nginx/error_pages/400.html [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+        html,
+        body {
             height: 100%;
             margin: 0;
-            font-family: Inter;
+            box-sizing: border-box;
+            font-family: Inter, sans-serif;
+            color: #eeeeee;
         }
+
```
Comment (pfaffkam) [inline nginx/error_pages/400.html]: Unnecessary empty lines between blocks (multiple occurencies)

File: nginx/error_pages/400.html [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-    <title>403</title>
+    <title><!--#echo var="status" --></title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;800&family=Manrope:wght@200;300;400;500;600;700;800&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <style>
-        html, body {
+        html,
+        body {
```
Comment (pfaffkam) [inline nginx/error_pages/400.html:10]: It can be in one line.

File: nginx/error_pages/400.html [hunk 3]
```diff
@@ -55,6 +110,53 @@ <h1><!--#echo var="status" --></h1>
...
+                    <tr>
+                        <td>Document root:</td><td><i><!--#echo var="document_root" --></i></td>
+                    </tr>
+                    <tr>
+                        <td>NGINX Version:</td><td><i><!--#echo var="nginx_version" --></i></td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+        <!--$endif -->
```
Comment (pfaffkam) [inline nginx/error_pages/400.html]:
```suggestion
        <!--#endif -->
```

File: nginx/error_pages/400.html [hunk 4]
```diff
@@ -55,6 +110,53 @@ <h1><!--#echo var="status" --></h1>
...
+            icon.classList.remove('arrow-down');
+            icon.classList.add('arrow-up');
+            content.style.opacity = 0;
+            content.style.height = 0;
+            content.style.padding = '0px';
+        }
+        isExpanded = !isExpanded;
+    });
+</script>
+<!--$endif -->
```
Comment (pfaffkam) [inline nginx/error_pages/400.html]:
```suggestion
<!--#endif -->
```

**Review by pfaffkam — CHANGES_REQUESTED** [review]

File: nginx/error_pages/400.html (hunk 1 above)
Comment (Kamil-Drozdz) [reply nginx/error_pages/400.html]: this is the default prettier setting, there is no custom setting in the project.

File: nginx/error_pages/400.html (hunk 2 above)
Comment (Kamil-Drozdz) [reply nginx/error_pages/400.html:10]: this is the default prettier setting, there is no custom setting in the project.

File: nginx/error_pages/400.html (hunk 3 above)
Comment (Kamil-Drozdz) [reply nginx/error_pages/400.html]: done.

File: nginx/error_pages/400.html (hunk 4 above)
Comment (Kamil-Drozdz) [reply nginx/error_pages/400.html]: done

File: nginx/error_pages/400.html (hunk 1 above)
Comment (pfaffkam) [reply nginx/error_pages/400.html]: Ok, I will fix it on my own.

File: nginx/error_pages/400.html (hunk 2 above)
Comment (pfaffkam) [reply nginx/error_pages/400.html:10]: Ok, I will fix it on my own.

**Pull request merged**